### DISCUSSION TITLE
inject text strings

### DISCUSSION THREAD

**Discussion by lucentwolf** [sezanzeb/input-remapper#864] [Q&A]
Hi All

I did not figure out how I would insert whole text strings without having to send every single letter as a key stroke; e.g. string("Abc") or text("Abc") instead of modify(KEY_LEFTSHIFT, key(a)).key(b).key(c) 

Am I missing something or would this be a feature request?

-^.^-

[EDIT: Typo]

**Comment by sabudum** [comment]
You could create a command that would read a text file and insert it into your clipboard, then inject Ctrl+V to paste it. All from a single macro.

**Reply by tummydummy** [reply]
Is there any chance you could provide more info on how to do this? I'm finding the documentation to be a bit different from whaat I'm looking for as a newbie. i.e. How to create a command to read a text file and copy it to the clipboard and then paste it.
